tensor_api.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<p>More descriptions are needed. See <a href="https://numpy.org/doc/stable/reference/arrays.indexing.html#advanced-indexing" target="_blank">https://numpy.org/doc/stable/reference/arrays.indexing.html#advanced-indexing</a> for details.</p><p>Descriptions should cover:</p><ul><li>What can be used as index for each dimension, and how they can be mixed e.g.</li><li class="ql-indent-1">integer (slicing)</li><li class="ql-indent-1">a slice object (slicing)</li><li class="ql-indent-1">an index array (triggers advanced indexing)</li><li class="ql-indent-1">a boolean array (triggers advanced indexing)</li><li>The getitem returns a copy, not a view, which is different from slicing</li><li>In particular, write about the followings and give examples(<a href="https://numpy.org/doc/stable/reference/arrays.indexing.html#advanced-indexing" target="_blank">https://numpy.org/doc/stable/reference/arrays.indexing.html#advanced-indexing</a> as a refernce)</li><li class="ql-indent-1">Purely integer array indexing</li><li class="ql-indent-1">Combining advanced and basic indexing, where there are two cases (write about how they work, how the output dimension is determined and give examples)</li><li class="ql-indent-2">The advanced indexes are separated by a slice, Ellipsis or newaxis.&nbsp;</li><li class="ql-indent-2">The advanced indexes are all next to each other.&nbsp;</li><li>Add the boolean indexing part, similar to the numpy example.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/intel-isl/Open3D/pull/2067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>